tests/unit/test_clidocs.py

```diff
@@ -613,3 +573,43 @@ def test_description_tags_in_body(self):
         self.doc_handler.doc_description(self.cmd)
         contents = self.cmd.doc.getvalue().decode('utf-8')
         self.assertIn(ref_body, contents)
+
+
+class TestGlobalOptionsDocumenter(unittest.TestCase):
+    def create_help_command(self):
+        types = ['blob', 'integer', 'boolean', 'string']
+        arg_table = {}
+        for t in types:
+            name = f'{t}_type'
+            help_text = f'This arg type is {t}'
+            choices = ['A', 'B', 'C'] if t == 'string' else []
+            arg_table[name] = CustomArgument(name=name,
+                                             cli_type_name=t,
+                                             help_text=help_text,
+                                             choices=choices)
+        help_command = mock.Mock()
```

Comment:
I mocked the `help_command` because in order to initialize an actual help command object I would need to mock out a `session` anyways. I added a `spec`.